### ISSUE TITLE
Feature: Display all network folders on the sidebar

### ISSUE THREAD

**Issue by iNether** [files-community/Files#8304]
### What's the Problem?

Network drives folder

### Solution/Idea

List of available network drives (PC's) instead of folder with all of them.

Now:
![NowDrives](https://user-images.githubusercontent.com/20440443/153609188-61f5dc01-52c5-4e65-8a7c-e0ea3560df8d.png)

Should be:
![networkDrives](https://user-images.githubusercontent.com/20440443/153592966-d8e6c33e-d1b2-4672-bcec-4c397ec2453a.png)
 

### Alternatives

.

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |

**Comment by iNether** [comment]
> Same as #8287

no it's not. I have access to the drives

**Comment by Josh-65** [comment]
from your issue it seemed you couldnt access the network computer shown in file explorer witch is the same as the title of the other one  

> Network Explorer can't access to other computers

**Comment by iNether** [comment]
> from your issue it seemed you couldnt access the network computer shown in file explorer witch is the same as the title of the other one. although there issue is not very desiptive.
> 
> > Network Explorer can't access to other computers

Read carefully. Devices from the network are nested in the "Network" folder, and are not immediately displayed in the menu

**Comment by Josh65-2201** [comment]
Asking @yair100 for approval.

**Comment by Fefedu973** [comment]
Same issue

**Comment by jaseinatl** [comment]
My report was inappropriately merged with this issue. My Network Drives panel option shows the network folder, but nothing in it. In Windows Explorer, I can see all of the network devices. In Files App, I see nothing but an empty folder. This ticket seems to be complaining that they have to open the Network folder inside the Network item in the panel. That's not my case at all. Whoever is managing these requests and closing them is not addressing the issue here.

**Comment by Josh65-2201** [comment]
The network folder scan for all items on the network but can be slow (There's a loading icon for this in the tab bar). Thats a limitation of Windows/File explorer.
If you see the main issue screenshot you can see this issue asks to show devices on the sidebar as well, which is what you are requesting.

**Comment by Lukiluc29** [comment]
Do you have any update on this feature request

**Comment by jaseinatl** [comment]
As far as I am concerned. feel free to close this ticket. After multiple
occurrences of my computer literally grinding to a halt while trying to
save or open files, I realized the one common denominator was Files. I
loved how your app looked, but it was impossibly slow in my configuration.
So slow, in fact, that I was able to reboot faster than it would have taken
to get a directory listing in Files from Office or any of the Adobe apps.
I wish you the best of luck, but it's just not there for me at this time.
Which sucks because Windows Explorer is a complete shitshow on my PC. I
have about 8 drives and half a dozen Operating Sysems installed so Windows
never knows where anything is. So t's not your fault, it's just not
compatible with my mess.
